diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/files/NewlinesRuleWarnTest.kt

```diff
@@ -502,4 +502,73 @@ class NewlinesRuleWarnTest : LintTestBase(::NewlinesRule) {
                 LintError(2, 5, ruleId, singleReturnWarn, true)
         )
     }
+
+    @Test
+    @Tag(WarningNames.WRONG_NEWLINES)
+    fun `should not trigger`() {
+        lintMethod(
+                """
+                    |fun foo(): String {
+                    |
+                    |       val a = java.lang.Boolean.getBoolean(properties.getProperty("parallel.mode"))
+                    |
+                    |        allProperties?.filter {
+                    |           predicate(it)
+                    |           val x = listOf(1,2,3).filter { it < 3 }
+                    |           x == 0
+                    |        }
+                    |        .foo()
+                    |        .bar()
+                    |        
+                    |        allProperties?.filter {
+                    |           predicate(it)
+                    |        }
+                    |        .foo()
+                    |        .bar()
+                    |        .let {
+                    |           it.some()
+                    |        }
+                    |        
+                    |        allProperties
+                    |        ?.filter {
+                    |           predicate(it)
+                    |        }
+                    |        .foo()
+                    |        .bar()
+                    |        .let {
+                    |           mutableListOf().also {
+                    |               
+                    |           }
+                    |        }
+                    |}
+                """.trimMargin()
+        )
+    }
+
+    @Test
+    @Tag(WarningNames.WRONG_NEWLINES)
+    fun `should trigger on non-multiline lambdas`() {
```

Comment:
Please add example like
```kotlin
list.filter {

}
.map(::foo).filter {
    bar()
}
```
It should be considered incorrect, when multiline lambda is not the first call.